api/lib/database/methods.js

```diff
@@ -1,5 +1,32 @@
 const { read, insert, query } = require('./helpers')
 
+const getAccountsNotifications = ({
+  hasura_url,
+  hasura_admin_key
+}) => () => async (addresses, networkId) => {
+  return await read({
+    hasura_url,
+    hasura_admin_key
+  })()(
+    `notifications`,
+    `notifications`,
+    [
+      'topic',
+      'eventType',
+      'resourceType',
+      'resourceId',
+      'networkId',
+      'data',
+      'id',
+      'created_at'
+    ],
+    `where: {
+      networkId: {_eq: "${networkId}"},
+      resourceId: {_in: [${addresses.map((address) => `"${address}"`)}]},
+    } limit: 10, order_by: {created_at: desc}`
```

Comment:
```suggestion
    }, order_by: {created_at: desc}`
```

can't limit here as you would only get 10 in total

Reply:
the original idea was to get latest notifications but yeah, we can change that. Or slice in FE, for sure

Reply:
OK, removed :+1: